### DISCUSSION TITLE
Installed via Pipx method can't find config.yml after 1st run

### DISCUSSION THREAD

**Discussion by CrocodileDuty** [Taxel/PlexTraktSync#1499] [Support] [answered]
Hello, I 1st tried the git clone method on my ubuntu machine but not knowing how to execute the script from there I tried Pipx method.. The script installed easily but I can't find config.yml under home/user/.local/Pipx/venvs/plextraktsync.. I can only find the config.default.yml there...

Any help in this regard will be appreciated
Thankyou

**Comment by simonc56** [comment]
Did you manage to setup Plex and Trakt access at first run ?
Show here the log output of first run.

**Reply by CrocodileDuty** [reply, marked as answer]
I was confused during running but now its working fine af.. Hence closed the discussion.

**Reply by simonc56** [reply]
Explain what you did to make it work.

**Reply by CrocodileDuty** [reply]
> Explain what you did to make it work.

I dropped the Pipx method and instead cloned the repo.. ran python3 -m plextraktsync and then it ran and created a config.yml file.
